Tasks/AzureAppServiceManageV0/azureappservicemanage.ts

```diff
@@ -64,6 +73,11 @@ async function run() {
         var endpointTelemetry = '{"endpointId":"' + connectedServiceName + '"}';
         console.log("##vso[telemetry.publish area=TaskEndpointId;feature=AzureAppServiceManage]" + endpointTelemetry);
         
+        if (azureEndpoint.scheme && azureEndpoint.scheme.toLowerCase() === AzureRmEndpointAuthenticationScheme.PublishProfile) {
+            await handleActionWithPublishProfileEndpoint(azureEndpoint, action, extensionList, extensionOutputVariables);
```

Comment:
ideally we should only try to throw here if action is not supported by publish profile. The code to handle action should be common.